mungegithub/mungers/blunderbuss.go

```diff
@@ -112,13 +100,13 @@ func (b *BlunderbussMunger) Munge(obj *github.MungeObject) {
 		// makes three buckets, we shouldn't have many 10k+
 		// line changes.
 		fileWeight = int64(math.Log10(float64(fileWeight))) + 1
-		fileOwners := b.features.Repos.LeafAssignees(*file.Filename)
+		fileOwners := feats.Repos.LeafAssignees(*file.Filename)
```

Comment:
@apelisse @foxish Depending on how the owners files are implemented, this could be problematic.  If people appear only at their highest level or only in one group (e.g. Brian, Tim only appear the root and only in the reviewers XOR approvers) then those kinds of people will never be able to self assign.  Ideas for handling this?